### PR TITLE
Add info on if a youtuber covers 3D or 2D content and what programming language they use

### PR DESCRIPTION
**Pull request by skyace65**
On the tutorials and resources page I've added information next to every youtube link on if a youtuber covers 2D content, 3D content or both, I also added what programming language they use. I realize this isn't perfect. Some youtubers cover significantly more 2D than 3D or more 3D than 2D, the same is true with programming languages. That said I think this is better than what we have now and I'm open to suggestions.

### CODE REVIEW

File: community/tutorials.rst [hunk 1]
```diff
@@ -19,20 +19,20 @@ Some of the others mentioned below provide more advanced tutorials, e.g. on 3D o
...
-- `P1X / Krzysztof Jankowski <https://www.youtube.com/playlist?list=PLvDk7UKhld4xGPovdB4IFtAHYMYjx_-3K>`_
-- `Pigdev <https://www.youtube.com/playlist?list=PLPMN4vCRFdordS3E-3zi0Hdh7pAsbWQ6a>`_
-- `Steincodes <https://www.youtube.com/c/steincodes/playlists>`__
-- `Code with Tom <https://www.youtube.com/playlist?list=PLiUQR4U_J9ec0k91iHPme_qtfS1nrWF3W>`_
-- `BornCG <https://www.youtube.com/playlist?list=PLda3VoSoc_TSBBOBYwcmlamF1UrjVtccZ>`_
+- `Emilio <https://www.youtube.com/channel/UC9DR22-qohBDtZ74R3FxOZg>`_ (2D, GDScript and Visual Scripting).
+- `GDQuest <https://www.youtube.com/channel/UCxboW7x0jZqFdvMdCFKTMsQ/playlists>`_ (2D and 3D, GDScript, VisualScripting and C# ).
+- `Game Endeavor <https://www.youtube.com/channel/UCLweX1UtQjRjj7rs_0XQ2Eg/videos>`_ (2D, GDScript).
+- `Game from Scratch <https://www.youtube.com/watch?v=iDEcP8Mc-7s&list=PLS9MbmO_ssyDk79j9ewONxV88fD5e_o5d>`_ (2D and 3D, GDScript and C#).
+- `HeartBeast <https://www.youtube.com/watch?v=wETY5_9kFtA&list=PL9FzW-m48fn2jlBu_0DRh7PvAt-GULEmd>`_ (2D, GDscript).
```
Comment (clayjohn) [inline community/tutorials.rst]: GDscript -> GDScript

File: community/tutorials.rst [hunk 2]
```diff
@@ -19,20 +19,20 @@ Some of the others mentioned below provide more advanced tutorials, e.g. on 3D o
...
+- `Jeremy Bullock <https://www.youtube.com/channel/UCwJw2-V5S1TkBjLQ3_Ws54g>`_ (2D and 3D, GDscript, Visual Scripting and C#).
```
Comment (clayjohn) [inline community/tutorials.rst]: GDscript -> GDScript

File: community/tutorials.rst [hunk 3]
```diff
@@ -19,20 +19,20 @@ Some of the others mentioned below provide more advanced tutorials, e.g. on 3D o
...
+- `KidsCanCode <https://www.youtube.com/channel/UCNaPQ5uLX5iIEHUCLmfAgKg/playlists>`__ (2D and 3D, Gdscript and Visual Scripting).
+- `Mister Taft Creates <https://www.youtube.com/playlist?list=PL4vbr3u7UKWqwQlvwvgNcgDL1p_3hcNn2>`_ (2D, GDscript).
```
Comment (clayjohn) [inline community/tutorials.rst]: GDscript -> GDScript

File: community/tutorials.rst [hunk 4]
```diff
@@ -19,20 +19,20 @@ Some of the others mentioned below provide more advanced tutorials, e.g. on 3D o
...
+- `Miziziziz <https://www.youtube.com/playlist?list=PLmugv6_kd0qN6AyjG245_Pdak4MXKUx88>`_ (2D, GDscript).
```
Comment (clayjohn) [inline community/tutorials.rst]: GDscript -> GDScript

File: community/tutorials.rst [hunk 5]
```diff
@@ -19,20 +19,20 @@ Some of the others mentioned below provide more advanced tutorials, e.g. on 3D o
...
+- `P1X / Krzysztof Jankowski <https://www.youtube.com/playlist?list=PLvDk7UKhld4xGPovdB4IFtAHYMYjx_-3K>`_ (3D).
+- `Pigdev <https://www.youtube.com/playlist?list=PLPMN4vCRFdordS3E-3zi0Hdh7pAsbWQ6a>`_ (2D, GDscript).
```
Comment (clayjohn) [inline community/tutorials.rst]: GDscript -> GDScript

**Comment by skyace65** [comment]
That should be fixed now.

File: community/tutorials.rst [hunk 6]
```diff
@@ -19,20 +19,20 @@ Some of the others mentioned below provide more advanced tutorials, e.g. on 3D o
...
-- `KidsCanCode <https://www.youtube.com/channel/UCNaPQ5uLX5iIEHUCLmfAgKg/playlists>`__
-- `Mister Taft Creates <https://www.youtube.com/playlist?list=PL4vbr3u7UKWqwQlvwvgNcgDL1p_3hcNn2>`_
-- `Miziziziz <https://www.youtube.com/playlist?list=PLmugv6_kd0qN6AyjG245_Pdak4MXKUx88>`_
-- `P1X / Krzysztof Jankowski <https://www.youtube.com/playlist?list=PLvDk7UKhld4xGPovdB4IFtAHYMYjx_-3K>`_
-- `Pigdev <https://www.youtube.com/playlist?list=PLPMN4vCRFdordS3E-3zi0Hdh7pAsbWQ6a>`_
-- `Steincodes <https://www.youtube.com/c/steincodes/playlists>`__
-- `Code with Tom <https://www.youtube.com/playlist?list=PLiUQR4U_J9ec0k91iHPme_qtfS1nrWF3W>`_
-- `BornCG <https://www.youtube.com/playlist?list=PLda3VoSoc_TSBBOBYwcmlamF1UrjVtccZ>`_
+- `Emilio <https://www.youtube.com/channel/UC9DR22-qohBDtZ74R3FxOZg>`_ (2D, GDScript and Visual Scripting).
+- `GDQuest <https://www.youtube.com/channel/UCxboW7x0jZqFdvMdCFKTMsQ/playlists>`_ (2D and 3D, GDScript, VisualScripting and C# ).
```
Comment (aaronfranke) [inline community/tutorials.rst]: `Visual Scripting` and `Visual Scripting` should be `VisualScript`.

You have an extra space before that closing parenthesis.

**Comment by skyace65** [comment]
Those should be fixed now.

**Comment by Calinou** [comment]
This needs a rebase before it can be merged. Other than that, it looks good to me :slightly_smiling_face:

**Comment by aaronfranke** [comment]
Squashed-and-merged. Thanks!

**Pull request merged**